recipes/full_finetune_distributed.py

```diff
@@ -131,6 +137,7 @@ def __init__(self, cfg: DictConfig) -> None:
         self.total_epochs = cfg.epochs
         self.max_steps_per_epoch = cfg.max_steps_per_epoch
         self.global_step = 0
+        self._clip_grad_norm = cfg.get("clip_grad_norm", None)
```

Comment:
Can we add a test case similar to [this](https://github.com/pytorch/torchtune/blob/9f329a261fce1935b40029914e38ee31d952c50a/tests/recipes/test_full_finetune_single_device.py#L49)?